### ISSUE TITLE
version 1.17.5 does not compile on iOS

### ISSUE THREAD

**Issue by hadi-neon** [cleverpush/cleverpush-flutter-sdk#33]
After upgrading to version 1.17.5 the app does not run 
here is the log:
<details>
  <summary>Log</summary>
  
Xcode build done.                                           14.7s
Failed to build iOS app
Error output from Xcode build:
↳
    objc[73055]: Class AppleTypeCRetimerRestoreInfoHelper is implemented in both /usr/lib/libauthinstall.dylib (0x200e17620) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1047d4598). One of the two will be used. Which one is undefined.
    objc[73055]: Class AppleTypeCRetimerFirmwareAggregateRequestCreator is implemented in both /usr/lib/libauthinstall.dylib (0x200e17670) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1047d45e8). One of the two will be used. Which one is undefined.
    objc[73055]: Class AppleTypeCRetimerFirmwareRequestCreator is implemented in both /usr/lib/libauthinstall.dylib (0x200e176c0) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1047d4638). One of the two will be used. Which one is undefined.
    objc[73055]: Class ATCRTRestoreInfoFTABFile is implemented in both /usr/lib/libauthinstall.dylib (0x200e17710) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1047d4688). One of the two will be used. Which one is undefined.
    objc[73055]: Class AppleTypeCRetimerFirmwareCopier is implemented in both /usr/lib/libauthinstall.dylib (0x200e17760) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1047d46d8). One of the two will be used. Which one is undefined.
    objc[73055]: Class ATCRTRestoreInfoFTABSubfile is implemented in both /usr/lib/libauthinstall.dylib (0x200e177b0) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1047d4728). One of the two will be used. Which one is undefined.
    2022-10-14 17:47:14.502 xcodebuild[73055:823597] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    2022-10-14 17:47:14.503 xcodebuild[73055:823597] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    ** BUILD FAILED **
Xcode's output:
↳
    Writing result bundle at path:
    	/var/folders/x6/fgz6wfl909nc3pw3zpfnmhyc0000gp/T/flutter_tools.3D2s6c/flutter_ios_build_temp_dirpQg9z5/temporary_xcresult_bundle
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/in_app_review-2.0.4/ios/Classes/InAppReviewPlugin.m:91:14: warning: 'openURL:' is deprecated: first deprecated in iOS 10.0 [-Wdeprecated-declarations]
            [app openURL:url];
                 ^~~~~~~
                 openURL:options:completionHandler:
    In module 'UIKit' imported from /Users/hadi/dev/2210_Ippen_App/ios/Pods/Target Support Files/in_app_review/in_app_review-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS15.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:124:1: note: 'openURL:' has been explicitly marked deprecated here
    - (BOOL)openURL:(NSURL*)url API_DEPRECATED_WITH_REPLACEMENT("openURL:options:completionHandler:", ios(2.0, 10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    1 warning generated.
    /* com.apple.ibtool.document.warnings */
    /Users/hadi/dev/2210_Ippen_App/ios/Pods/CleverPush/CleverPush/Resources/CPBannerCardContainer.xib:i8s-pt-aNt: warning: Estimated section header height before iOS 11.0 [5]
    /Users/hadi/dev/2210_Ippen_App/ios/Pods/CleverPush/CleverPush/Resources/CPBannerCardContainer.xib:i8s-pt-aNt: warning: Estimated section footer height before iOS 11.0 [5]
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:171:50: warning: 'sdkInfo' is deprecated: This property will be removed in a future version of the SDK
                if let sdkInfo = self.sentryOptions?.sdkInfo {
                                                     ^
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:470:35: warning: expression implicitly coerced from 'Dictionary<String, Any?>' to '[String : Any]'
              scope.setContext(value: dictionary, key: key)
                                      ^~~~~~~~~~
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:470:35: note: explicitly cast to '[String : Any]' with 'as [String : Any]' to silence this warning
              scope.setContext(value: dictionary, key: key)
                                      ^~~~~~~~~~
                                                 as [String : Any]
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:472:45: warning: expression implicitly coerced from 'String?' to 'Any'
              scope.setContext(value: ["value": string], key: key)
                                                ^~~~~~
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:472:45: note: provide a default value to avoid this warning
              scope.setContext(value: ["value": string], key: key)
                                                ^~~~~~
                                                       ?? <#default value#>
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:472:45: note: force-unwrap the value to avoid this warning
              scope.setContext(value: ["value": string], key: key)
                                                ^~~~~~
                                                      !
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:472:45: note: explicitly cast to 'Any' with 'as Any' to silence this warning
              scope.setContext(value: ["value": string], key: key)
                                                ^~~~~~
                                                       as Any
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:512:31: warning: expression implicitly coerced from 'Dictionary<String, Any?>' to '[String : Any]'
              userInstance.data = extras
                                  ^~~~~~
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:512:31: note: explicitly cast to '[String : Any]' with 'as [String : Any]' to silence this warning
              userInstance.data = extras
                                  ^~~~~~
                                         as [String : Any]
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:557:35: warning: expression implicitly coerced from 'Dictionary<String, Any?>' to '[String : Any]'
            breadcrumbInstance.data = data
                                      ^~~~
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/sentry_flutter-6.9.1/ios/Classes/SentryFlutterPluginApple.swift:557:35: note: explicitly cast to '[String : Any]' with 'as [String : Any]' to silence this warning
            breadcrumbInstance.data = data
                                      ^~~~
                                           as [String : Any]
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Types/URLCredential.swift:14:25: warning: comparing non-optional value of type '[Any]' to 'nil' always returns true
            if certificates != nil {
               ~~~~~~~~~~~~ ^  ~~~
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:86:25: warning: 'spotlightSuggestion' was deprecated in iOS 10.0: renamed to 'WKDataDetectorTypes.lookupSuggestion'
                    return .spotlightSuggestion
                            ^
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:86:25: note: use 'WKDataDetectorTypes.lookupSuggestion' instead
                    return .spotlightSuggestion
                            ^~~~~~~~~~~~~~~~~~~
                            WKDataDetectorTypes.lookupSuggestion
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:121:31: warning: 'spotlightSuggestion' was deprecated in iOS 10.0: renamed to 'WKDataDetectorTypes.lookupSuggestion'
                if type.contains(.spotlightSuggestion) {
                                  ^
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:121:31: note: use 'WKDataDetectorTypes.lookupSuggestion' instead
                if type.contains(.spotlightSuggestion) {
                                  ^~~~~~~~~~~~~~~~~~~
                                  WKDataDetectorTypes.lookupSuggestion
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:216:21: warning: immutable value 'j' was never used; consider replacing with '_' or removing it
                    for j in ipv6.count...8 {
                        ^
                        _
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Types/URLCredential.swift:14:25: warning: comparing non-optional value of type '[Any]' to 'nil' always returns true
            if certificates != nil {
               ~~~~~~~~~~~~ ^  ~~~
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:86:25: warning: 'spotlightSuggestion' was deprecated in iOS 10.0: renamed to 'WKDataDetectorTypes.lookupSuggestion'
                    return .spotlightSuggestion
                            ^
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:86:25: note: use 'WKDataDetectorTypes.lookupSuggestion' instead
                    return .spotlightSuggestion
                            ^~~~~~~~~~~~~~~~~~~
                            WKDataDetectorTypes.lookupSuggestion
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:121:31: warning: 'spotlightSuggestion' was deprecated in iOS 10.0: renamed to 'WKDataDetectorTypes.lookupSuggestion'
                if type.contains(.spotlightSuggestion) {
                                  ^
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:121:31: note: use 'WKDataDetectorTypes.lookupSuggestion' instead
                if type.contains(.spotlightSuggestion) {
                                  ^~~~~~~~~~~~~~~~~~~
                                  WKDataDetectorTypes.lookupSuggestion
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/Util.swift:216:21: warning: immutable value 'j' was never used; consider replacing with '_' or removing it
                    for j in ipv6.count...8 {
                        ^
                        _
    /Users/hadi/.pub-cache/git/flutter_inappwebview-252d836d52bff32870ed1b72231e428b0009e89e/ios/Classes/InAppBrowser/InAppBrowserWebViewController.swift:561:13: warning: setter for 'statusBarStyle' was deprecated in iOS 9.0: Use -[UIViewController preferredStatusBarStyle]
                UIApplication.shared.statusBarStyle = UIStatusBarStyle(rawValue: previousStatusBarStyle)!
                ^
    Command CompileSwiftSources failed with a nonzero exit code
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.7/ios/Classes/ReachabilityConnectivityProvider.swift:23:5: warning: result of call to 'ensureReachability()' is unused
        ensureReachability()
        ^                 ~~
In file included from /Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.m:1:
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:4:4: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
    - (instancetype)initWithMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
       ^
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:4:4: note: insert '_Nullable' if the pointer may be null
    - (instancetype)initWithMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
       ^
                    _Nullable
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:4:4: note: insert '_Nonnull' if the pointer should never be null
    - (instancetype)initWithMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
       ^
                    _Nonnull
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:11:64: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
                  binaryMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
                                                                   ^
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:11:64: note: insert '_Nullable' if the pointer may be null
                  binaryMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
                                                                   ^
                                                                    _Nullable
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:11:64: note: insert '_Nonnull' if the pointer should never be null
                  binaryMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
                                                                   ^
                                                                    _Nonnull
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:13:10: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
    - (UIView*)view;
             ^
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:13:10: note: insert '_Nullable' if the pointer may be null
    - (UIView*)view;
             ^
              _Nullable
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:13:10: note: insert '_Nonnull' if the pointer should never be null
    - (UIView*)view;
             ^
              _Nonnull
While building module 'CleverPush' imported from /Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CleverPushPlugin.h:2:
    In file included from <module-includes>:1:
    In file included from /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPush.h:20:
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPushInstance.h:45:37: warning: this block declaration is not a prototype [-Wstrict-prototypes]
    typedef void (^CPTopicsChangedBlock)();
                                        ^
                                         void
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPushInstance.h:140:68: warning: this block declaration is not a prototype [-Wstrict-prototypes]
    - (void)showTopicsDialog:(UIWindow *)targetWindow callback:(void(^)())callback;
                                                                       ^
                                                                        void
While building module 'CleverPush' imported from /Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CleverPushPlugin.h:2:
    In file included from <module-includes>:1:
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPush.h:103:68: warning: this block declaration is not a prototype [-Wstrict-prototypes]
    + (void)showTopicsDialog:(UIWindow *)targetWindow callback:(void(^)())callback;
                                                                       ^
                                                                        void
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPush.h:169:1: warning: umbrella header for module 'CleverPush' does not include header 'CPWKWebKitView.h' [-Wincomplete-umbrella]
    4 warnings generated.
    7 warnings generated.
While building module 'CleverPush' imported from /Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CleverPushPlugin.h:2:
    In file included from <module-includes>:1:
    In file included from /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPush.h:20:
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPushInstance.h:45:37: warning: this block declaration is not a prototype [-Wstrict-prototypes]
    typedef void (^CPTopicsChangedBlock)();
                                        ^
                                         void
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPushInstance.h:140:68: warning: this block declaration is not a prototype [-Wstrict-prototypes]
    - (void)showTopicsDialog:(UIWindow *)targetWindow callback:(void(^)())callback;
                                                                       ^
                                                                        void
While building module 'CleverPush' imported from /Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CleverPushPlugin.h:2:
    In file included from <module-includes>:1:
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPush.h:103:68: warning: this block declaration is not a prototype [-Wstrict-prototypes]
    + (void)showTopicsDialog:(UIWindow *)targetWindow callback:(void(^)())callback;
                                                                       ^
                                                                        void
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPush.h:169:1: warning: umbrella header for module 'CleverPush' does not include header 'CPWKWebKitView.h' [-Wincomplete-umbrella]
    4 warnings generated.
In file included from /Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CleverPushPlugin.m:2:
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:4:4: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
    - (instancetype)initWithMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
       ^
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:4:4: note: insert '_Nullable' if the pointer may be null
    - (instancetype)initWithMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
       ^
                    _Nullable
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:4:4: note: insert '_Nonnull' if the pointer should never be null
    - (instancetype)initWithMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
       ^
                    _Nonnull
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:11:64: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
                  binaryMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
                                                                   ^
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:11:64: note: insert '_Nullable' if the pointer may be null
                  binaryMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
                                                                   ^
                                                                    _Nullable
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:11:64: note: insert '_Nonnull' if the pointer should never be null
                  binaryMessenger:(NSObject<FlutterBinaryMessenger>*)messenger;
                                                                   ^
                                                                    _Nonnull
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:13:10: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
    - (UIView*)view;
             ^
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:13:10: note: insert '_Nullable' if the pointer may be null
    - (UIView*)view;
             ^
              _Nullable
/Users/hadi/.pub-cache/hosted/pub.dartlang.org/cleverpush_flutter-1.17.5/ios/Classes/CPChatViewFlutter.h:13:10: note: insert '_Nonnull' if the pointer should never be null
    - (UIView*)view;
             ^
              _Nonnull
    7 warnings generated.
    While building module 'CleverPush' imported from /Users/hadi/dev/2210_Ippen_App/ios/CleverPushNotificationServiceExtension/NotificationService.m:1:
    In file included from <module-includes>:1:
    In file included from /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPush.h:20:
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPushInstance.h:45:37: warning: this block declaration is not a prototype [-Wstrict-prototypes]
    typedef void (^CPTopicsChangedBlock)();
                                        ^
                                         void
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPushInstance.h:140:68: warning: this block declaration is not a prototype [-Wstrict-prototypes]
    - (void)showTopicsDialog:(UIWindow *)targetWindow callback:(void(^)())callback;
                                                                       ^
                                                                        void
    While building module 'CleverPush' imported from /Users/hadi/dev/2210_Ippen_App/ios/CleverPushNotificationServiceExtension/NotificationService.m:1:
    In file included from <module-includes>:1:
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPush.h:103:68: warning: this block declaration is not a prototype [-Wstrict-prototypes]
    + (void)showTopicsDialog:(UIWindow *)targetWindow callback:(void(^)())callback;
                                                                       ^
                                                                        void
    /Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/Headers/CleverPush.h:169:1: warning: umbrella header for module 'CleverPush' does not include header 'CPWKWebKitView.h' [-Wincomplete-umbrella]
    4 warnings generated.
    4 warnings generated.
    ld: '/Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/CleverPush' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file '/Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/CleverPush' for architecture arm64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in dependency order
    note: Removed stale file '/Users/hadi/Library/Developer/Xcode/DerivedData/Runner-djbnubgbfmxsbtgrpiiumayxhngc/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/GoogleAppMeasurement.build/InputFileList-F02B1DAD06931BC1E1CFAFA1DC051DE8-GoogleAppMeasurement-xcframeworks-input-files-f664d059b9c075974ef1b01ca50af404-resolved.xcfilelist'
    note: Removed stale file '/Users/hadi/Library/Developer/Xcode/DerivedData/Runner-djbnubgbfmxsbtgrpiiumayxhngc/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/GoogleAppMeasurement.build/OutputFileList-F02B1DAD06931BC1E1CFAFA1DC051DE8-GoogleAppMeasurement-xcframeworks-output-files-2aa8106d5b4f4276b9f414f6963d61aa-resolved.xcfilelist'
    note: Removed stale file '/Users/hadi/Library/Developer/Xcode/DerivedData/Runner-djbnubgbfmxsbtgrpiiumayxhngc/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/GoogleAppMeasurement.build/Script-F02B1DAD06931BC1E1CFAFA1DC051DE8.sh'
    note: Removed stale file '/Users/hadi/Library/Developer/Xcode/DerivedData/Runner-djbnubgbfmxsbtgrpiiumayxhngc/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/FirebaseAnalytics.build/InputFileList-4CAC89E59AEF75E870AB056400B23150-FirebaseAnalytics-xcframeworks-input-files-906190e1527a01598f772196d026d193-resolved.xcfilelist'
    note: Removed stale file '/Users/hadi/Library/Developer/Xcode/DerivedData/Runner-djbnubgbfmxsbtgrpiiumayxhngc/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/FirebaseAnalytics.build/OutputFileList-4CAC89E59AEF75E870AB056400B23150-FirebaseAnalytics-xcframeworks-output-files-3ee89396cd67ddec7321b0b57a08f4a3-resolved.xcfilelist'
    note: Removed stale file '/Users/hadi/Library/Developer/Xcode/DerivedData/Runner-djbnubgbfmxsbtgrpiiumayxhngc/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/FirebaseAnalytics.build/Script-4CAC89E59AEF75E870AB056400B23150.sh'
    /Users/hadi/dev/2210_Ippen_App/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.4.99. (in target 'ReachabilitySwift' from project 'Pods')
    /Users/hadi/dev/2210_Ippen_App/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.4.99. (in target 'OrderedSet' from project 'Pods')
    /Users/hadi/dev/2210_Ippen_App/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.4.99. (in target 'AppAuth' from project 'Pods')
    /Users/hadi/dev/2210_Ippen_App/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.4.99. (in target 'FMDB' from project 'Pods')
    /Users/hadi/dev/2210_Ippen_App/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.4.99. (in target 'CleverPush-CleverPushResources' from project 'Pods')
    /Users/hadi/dev/2210_Ippen_App/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.4.99. (in target 'CleverPush' from project 'Pods')
    Result bundle written to path:
    	/var/folders/x6/fgz6wfl909nc3pw3zpfnmhyc0000gp/T/flutter_tools.3D2s6c/flutter_ios_build_temp_dirpQg9z5/temporary_xcresult_bundle
Could not build the precompiled application for the device.

  
</details>

**Comment by gbhrdt** [comment]
Hi @hadi-neon  the issue seems to be that your Flutter project has Bitcode enabled:

```
ld: '/Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/CleverPush' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file '/Users/hadi/dev/2210_Ippen_App/build/ios/Debug-iphoneos/XCFrameworkIntermediates/CleverPush/CleverPush.framework/CleverPush' for architecture arm64
```

As we are using the newest iOS SDK with XCFramework support in the latest 1.17.5 version, our SDK does not come with bitcode support anymore, unfortunately. Also [Xcode 14 has deprecated bitcode submissions](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes).

Flutter will soon [drop bitcode support](https://github.com/flutter/flutter/issues/107887) officially.

You should already be able to disable bitcode manually in the Xcode build settings by setting `ENABLE_BITCODE` to `NO` in all targets.

**Comment by gbhrdt** [comment]
Closing, as issue seems to be resolved now :-).

**Comment by jmagman** [comment]
Flutter will handle setting `ENABLE_BITCODE` to `NO` for you when https://github.com/flutter/flutter/pull/112828 reaches a stable, it's currently available in 3.4.0-37.0.pre.

**Comment by gbhrdt** [comment]
Thanks @jmagman  for your update here, really helpful!
